xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -2756,7 +2756,7 @@ The property name should already be escaped when the instance of <xref:System.Te
         </remarks>
         <exception cref="T:System.InvalidOperationException">The depth of the JSON has exceeded the maximum depth of 1000.
     
-- or -
+-or-
```

Comment:
This was not showing up correctly:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonwriter.writestartarray?view=netcore-3.0#System_Text_Json_Utf8JsonWriter_WriteStartArray_System_Text_Json_JsonEncodedText_

![image](https://user-images.githubusercontent.com/6527137/64503362-87a61980-d27f-11e9-9f38-d34041109e01.png)